lib/ffi/gdal/internal_helpers/gdal_version.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module FFI
+  module GDAL
+    module InternalHelpers
+      # Version information for GDAL.
+      class GDALVersion
+        # @return [String] GDAL Version.
+        def self.version
+          ::FFI::GDAL.GDALVersionInfo("VERSION_NUM")
+        end
```

Comment:
This is already defined in `GDAL::VersionInfo` and extended by GDAL so you can `GDAL.version_num`.

Reply:
Yes, that's true. But we should require `gdal` to access `GDAL::VersionInfo`.

This might be unexpected for the library's end-user if he wants to require only FFI wrappers without Ruby-fied API wrappers.

From my point of view, the best case would be to have:
- FFI wrappers (that would be self-sufficient and provide only a lightweight FFI layer), so the client could require them from gem directly (e.q. `require ffi-gdal/ffi-gdal`);
- Ruby-fied API (e.q. `require ffi-gdal/gdal`) that will use FFI wrappers under the hood.

So, I have introduced a bit of duplication, but it feels like a reasonable tradeoff.